datafusion/datasource-parquet/src/source.rs

```diff
@@ -587,4 +578,17 @@ impl FileSource for ParquetSource {
             }
         }
     }
+
+    fn supports_dynamic_filter_pushdown(&self) -> bool {
+        true
+    }
```

Comment:
I guess I could nix this method and just treat a `Ok(None)` from `push_down_dynamic_filter` as not supported

Reply:
Okay will refactor to do this instead 👍🏻